app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def posts; end
+
+  def post; end
```

Comment:
- [ Optional] I would recommend you to use the `show` as an action to show all posts which can increase your code readability and not confuse other developers reading your code because it's the most popular naming convention.

Reply:
Thank you for the feedback. That must be done as you said. I fixed.🎉